lib/Db/ColumnTypes/DatetimeColumnQB.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace OCA\Tables\Db\ColumnTypes;
+
+use OCP\DB\QueryBuilder\IQueryBuilder;
+use Psr\Log\LoggerInterface;
+
+class DatetimeColumnQB extends SuperColumnQB implements IColumnTypeQB {
+	protected LoggerInterface $logger;
+	protected int $platform;
+
+	public function passSearchValue(IQueryBuilder &$qb, string $unformattedSearchValue, string $operator, string $searchValuePlaceHolder): void {
```

Comment:
```suggestion
	public function passSearchValue(IQueryBuilder $qb, string $unformattedSearchValue, string $operator, string $searchValuePlaceHolder): void {
```

Nitpick: objects are passed by references by default